data/migrations/20200426223901_music.js

```diff
@@ -17,6 +17,8 @@ exports.up = async function (knex, Promise) {
     video.increments();
     video.string("video_title").notNullable();
     video.string("location").notNullable();
+    video.boolean("videoCreated").defaultTo("false");
+    video.string("thumbnail");
```

Comment:
I like using a boolean, but we probably want to stick with our naming convention, something like `video_created` instead

Reply:
Good catch!

Reply:
updates the migration to match naming convention at this commit https://github.com/Lambda-School-Labs/artificial-artist-be/pull/23/commits/efdfcbdc5ee5831119b99b3d0bf09e26cd7a0a39